src/v0/destinations/hs/util.js

```diff
@@ -435,7 +435,7 @@ const getEventAndPropertiesFromConfig = (message, destination, payload) => {
   if (!hubspotEvents) {
     throw new InstrumentationError('Event and property mappings are required for track call');
   }
-  event = event.trim().toLowerCase();
+  event = String(event).trim().toLowerCase();
```

Comment:
add test cases and better to use function from utility https://github.com/rudderlabs/rudder-transformer/blob/main/src/v0/util/index.js

Reply:
addressed